### DISCUSSION TITLE
Bootstrap 5 checkboxes only works for line-height 1.5

### DISCUSSION THREAD

**Discussion by Jaqenhghar** [twbs/bootstrap#35855] [Q&A]
In the documentation here: 
https://getbootstrap.com/docs/5.0/forms/checks-radios/

it is easy to break the layout of all the checkboxes (and radios) in the page by giving the body a line-height of for example 2 (in firebug or similar).

I see the label is affected by the line-height, while the checkbox is not.

How can I get a consistent layout on different pages with different line-heights, while keeping the line-height on each page constant?

Thanks for any input.

**Comment by 035365** [comment]
[file.zip](https://github.com/twbs/bootstrap/files/8092226/file.zip)
